src/controller/bus.js

```diff
@@ -13,7 +13,7 @@ const createbus=async(req,res)=>{
        }
        else{
          const bus=await busservice.createbus(busdata)
-       res.status(201).json({bus:bus,"message":"bus created successfully"})
```

Comment:
"bus: bus" is not needed, you put "bus".
And when you are writing json where there are more than one element, delimit each element by newline.